traffic_portal/app/src/traffic_portal_properties.json

```diff
@@ -407,6 +407,12 @@
           "url": "http://trafficcontrol.apache.org/"
         }
       ]
+    },
+    "oAuth": {
+      "_comment": "Opt-in OAuth properties for SSO login",
+      "enabled": false,
+      "oAuthUrl": "",
```

Comment:
can you put some kind of fake example value in oAuthUrl and oAuthTokenQueryParam just to give us an idea of what to put in here. also, can you maybe explain a bit more in the _comment field?

Reply:
done